sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample10_RunningWithDifferentStorageVersion.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor.Samples.Infrastructure;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Storage.Blobs;
+
+/**
+ * The following sample can be used if the environment you are targeting supports a different version of Storage Blob SDK than those
```

Comment:
nit:  `Azure Storage Blobs` .... the storage team would like to be sure that the `s` exists on the end.